src/main/java/dev/goral/rpgmanager/rpgSystems/cpRed/characters/weapons/CpRedWeapons.java

```diff
@@ -0,0 +1,50 @@
+package dev.goral.rpgmanager.rpgSystems.cpRed.characters.weapons;
+
+import dev.goral.rpgmanager.rpgSystems.cpRed.characters.items.CpRedItemsAvailability;
+import dev.goral.rpgmanager.rpgSystems.cpRed.characters.skills.CpRedSkills;
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+public class CpRedWeapons {
+    @Id
+    @SequenceGenerator(
+            name = "cpRedWeapons_sequence",
+            sequenceName = "cpRedWeapons_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "cpRedWeapons_sequence"
+    )
+    private Long id;
+    @ManyToOne
+    @JoinColumn(
+            name="skill_id",
+            referencedColumnName = "id"
+    )
+    private CpRedSkills requireSkillId;
```

Comment:
```suggestion
    private CpRedSkills requiredSkillId;
```
nie miało być w ten sposób?